hs_core/views/serializers.py

```diff
@@ -0,0 +1,75 @@
+__author__ = 'Pabitra'
+
+from hs_core import hydroshare
+from django.contrib.auth.models import Group, User
+from rest_framework import serializers
+from .utils import authorize, validate_json, validate_user_name, validate_group_name
+from hs_core.hydroshare import utils
+
+
+class ResourceUpdateRequestValidator(serializers.Serializer):
+    title = serializers.CharField(required=False)
+    metadata = serializers.CharField(validators=[validate_json], required=False)
+    # TODO: use a custom validator for user names and group names
+    user_choices = [user.username for user in User.objects.all()]
+    group_choices = [grp.name for grp in Group.objects.all()]
+    edit_users = serializers.MultipleChoiceField(choices=user_choices, required=False)
```

Comment:
Multi-value field requires 'choices' attribute to be set. I guess if I create the list of user names outside this class I could set this attribute.